### PR TITLE
Small Bug Fix

### PR DESCRIPTION
**Pull request by yasmeen45**
## Description

Bug fix for project contact font and project side bad collapse button (was not clickable).

## Updates

Changed CSS for contact font on ProjectPeople.css and added margin for side bar collapse button.

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by yasmeen45** [comment]
[small bug fix](https://trello.com/c/2y9Mkal6/104-small-bug-fix)

File: package-lock.json
Comment (ElisaHu1) [inline package-lock.json:1]: you need to remove all the unnecessary change from packge-lock.json
I have compile error when run in your branch
